### DISCUSSION TITLE
How to install Hardware Key Driver from GitLab

### DISCUSSION THREAD

**Discussion by JesseHelm** [Kunzisoft/KeePassDX#1500] [General]
Im having trouble installing the driver, when I click on the GitLab download link and try to DL a package I get "Artifacts.zip".

Clearly its me not knowing how to do this, just wondering if anyone can help.

Thanks!

**Comment by J-Jamet** [comment]
Unzip the zip file and install the apk file that is in the folder. This is a debug version, autogenerated by gitlab.

**Reply by JesseHelm** [reply]
Thanks! I've got all the latest versions installed and the Key Driver. when I click 'open' it goes to the Key Driver with a pic of a Yubikey and loading animation but it always fails and brings me back to KeePassDX then about 3sec later I get a red bar that says" Unable to read credentials" No error logs. Im running a Pixel 6pro with GrapheneOS
Your help is much appreciated, thanks!

**Reply by J-Jamet** [reply]
Version 0.1.7 remained in debug mode with a test set, I solved the debug mode of version 0.1.8

**Reply by JesseHelm** [reply]
Right on! Thank you so much for your work and for taking the time to reply :)

**Reply by JesseHelm** [reply]
Just wanted to let you know it is working perfectly! Even the NFC! Great work!

**Comment by Schlats** [comment]
When I extract the zip file I see no apk file. I'm a new android user, so likely I'm missing something. I'm downloading the master file and using fx file manager. Help is much appreciated.

**Reply by Schlats** [reply]
Nevermind, I figured out to download from the package link.

**Reply by J-Jamet** [reply]
There is an APK file in the subfolders.
